rotkehlchen/api/rest.py

```diff
@@ -1314,7 +1314,7 @@ def user_premium_key_remove(self) -> Response:
         # else
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
-    def query_all_assets(self, filter_query: 'AssetsFilterQuery') -> Response:
+    def query_all_assets(self, filter_query: AssetsFilterQuery) -> Response:
```

Comment:
I believe is better to have those types as string if we don't need to use the class since they are only loaded during the linting and not actual code execution. I've checked and we do the same in other places of this file but maybe is worth keeping the string here and the other places where this filter is used

Reply:
it was a nitpick from a previous pr. The class is being used in this module.